packages/manager/apps/web/client/app/hosting/database/order/public/hosting-database-order-public.html

```diff
@@ -0,0 +1,34 @@
+<oui-back-button data-on-click=":: $ctrl.goBack()">
+    <span data-translate="ovhManagerHostingDatabaseOrderPublic_title"></span>
+</oui-back-button>
+
+<ovh-manager-product-offers
+    workflow-type="$ctrl.bindings.characteristics.workflowType"
+    workflow-options="$ctrl.bindings.characteristics.workflowOptions"
+    pricing-type="$ctrl.bindings.characteristics.pricingCapacity"
+    user=":: $ctrl.user"
+    on-success=":: $ctrl.onSuccess(checkout)"
+    on-error=":: $ctrl.onError(error)"
+    send-current-state=":: $ctrl.getOrderState(state)"
+>
+    <oui-step-form
+        header="{{:: 'ovhManagerHostingDatabaseOrderPublic_customization_title' | translate }}"
```

Comment:
Translation key casing is not really consistent

Reply:
Still waiting for you to answer my github issue about this ;)